We don't have the actionview gem in aports and it pulls some prebuilt binaries
when installed via bundler.

--- a/test/helper.rb
+++ b/test/helper.rb
@@ -9,15 +9,11 @@
 end
 Warning.prepend(HideOwnWarnings)
 
-require 'bundler/setup'
 require 'escape_utils'
 
 require 'active_support'
 require 'active_support/json'
 require "active_support/core_ext/string/output_safety"
-
-require 'action_view'
-require 'action_view/helpers'
 
 # bring in minitest
 require 'minitest/autorun'
